common/source/docs/common-remoteid.rst

```diff
@@ -26,17 +26,36 @@ Stand-alone devices:
 OpenDroneID
 ===========
 
+.. image:: ../../../images/remoteid.jpg
+    :target: ../../_images/remoteid.jpg
+
+
 ArduPilot provides support for OpenDroneID in firmware versions 4.2 and later via MAVLink to an OpenDroneID compatible module attached to either a serial port or DroneCAN port on the autopilot.
```

Comment:
Would it be clearer to say "flight controller"?

Reply:
no...autopilot is used throughout the wiki....many vehicles do not fly